ydb/core/tx/schemeshard/schemeshard_path_describer.cpp

```diff
@@ -1238,12 +1238,7 @@ void TSchemeShard::DescribeTableIndex(const TPathId& pathId, const TString& name
     const auto& tableStats = indexImplTable->GetStats().Aggregated;
     entry.SetDataSize(tableStats.DataSize + tableStats.IndexSize);
 
-    *entry.MutablePartitioningPolicy() = indexImplTable->PartitionConfig().GetPartitioningPolicy();
-    if (const auto& explicitPartitions = indexImplTable->TableDescription.GetSplitBoundary();
-        !explicitPartitions.empty()
-    ) {
-        *entry.MutableExplicitPartitions()->MutableSplitBoundary() = explicitPartitions;
-    }
+    *entry.AddIndexImplTableDescription() = indexImplTable->TableDescription;
```

Comment:
The entire TableDescription is copied here in the TIndexDescription. However, it does not contain a lot of info. Mainly, it does not contain TTablePartition which is heavy for multishard tables.

<img width="1728" alt="Screenshot 2024-07-04 at 12 12 30" src="https://github.com/ydb-platform/ydb/assets/111313089/99e21911-fa28-4cd5-986a-6117bdce7d4d">

Reply:
I wonder why there is no column info in the IndexImplTableDescription... I will solve this problem in scope of https://github.com/ydb-platform/ydb/issues/4955